### ISSUE TITLE
[Bug]: Filter sort option : (Sort by) only shows newest in the browser

### ISSUE THREAD

**Issue by Kaleeb44** [BlafKing/sd-civitai-browser-plus#225]
### Describe the bug.

When searching with an empty search field the browser only ever shows newest available models, using sort by: most downloaded, oldest, most liked, highest rated, Ect... the search results only show results from the newest option, and does not sort the models to the specified filter properly. search works fine, content type and base model also work as expected.

![Screenshot 2024-03-13 093708](https://github.com/BlafKing/sd-civitai-browser-plus/assets/163318717/4a8bfaee-f465-49ed-b81c-e31807668929)
![Screenshot 2024-03-13 093735](https://github.com/BlafKing/sd-civitai-browser-plus/assets/163318717/6e90670c-a3d5-458d-ad83-1a4625b0716d)
![Screenshot 2024-03-13 093814](https://github.com/BlafKing/sd-civitai-browser-plus/assets/163318717/1d1990cf-4a93-48de-9b33-872876ce22ad)

### Steps to reproduce the problem.

1. Go to CivitAI Browser plus
2. Press the filter button 
3. Select the desired filter under sort by
4. enter nothing in the search bar
5. press search button.

### Expected behavior

sort models based on desired field selected in the filter EG: most downloaded, highest rated, most buzz, most discussed, ect.

### System info

extention version: c947679f
"Platform": "Windows-10-10.0.19045-SP0",
"Python": "3.10.6",
"Version": "1.8.0-RC",

### Console logs

```Shell
there are no errors shown in the console when performing the test

venv "E:\Pro\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
WARNING: ZLUDA works best with SD.Next. Please consider migrating to SD.Next.
fatal: No names found, cannot describe anything.
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: 1.8.0-RC
Commit hash: 25a3b6cbeea8a07afd5e4594afc2f1c79f41ac1a
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
E:\Pro\stable-diffusion-webui-directml\venv\lib\site-packages\pytorch_lightning\utilities\distributed.py:258: LightningDeprecationWarning: `pytorch_lightning.utilities.distributed.rank_zero_only` has been deprecated in v1.8.1 and will be removed in v2.0.0. You can import it from `pytorch_lightning.utilities` instead.
  rank_zero_deprecation(
Launching Web UI with arguments: --use-zluda --skip-torch-cuda-test --gradio-auth caleb:1407 --autolaunch --disable-nan-check
E:\Pro\stable-diffusion-webui-directml\venv\lib\site-packages\diffusers\utils\outputs.py:63: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
E:\Pro\stable-diffusion-webui-directml\venv\lib\site-packages\diffusers\utils\outputs.py:63: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
ONNX: selected=CUDAExecutionProvider, available=['AzureExecutionProvider', 'CPUExecutionProvider']
[-] ADetailer initialized. version: 24.3.0, num models: 11
CivitAI Browser+: Aria2 RPC started

03/13 09:51:10 [NOTICE] IPv4 RPC: listening on TCP port 24000

03/13 09:51:10 [NOTICE] IPv6 RPC: listening on TCP port 24000
fatal: No names found, cannot describe anything.
ControlNet preprocessor location: E:\Pro\stable-diffusion-webui-directml\extensions\sd-webui-controlnet\annotator\downloads
2024-03-13 09:51:11,162 - ControlNet - INFO - ControlNet v1.1.441
2024-03-13 09:51:11,293 - ControlNet - INFO - ControlNet v1.1.441
Loading weights [ac3c546464] from E:\Pro\stable-diffusion-webui-directml\models\Stable-diffusion\realDelicious_v311.safetensors
Creating model from config: E:\Pro\stable-diffusion-webui-directml\configs\v1-inference.yaml
2024-03-13 09:51:11,824 - ControlNet - INFO - ControlNet UI callback registered.
Loading VAE weights from user metadata: E:\Pro\stable-diffusion-webui-directml\models\VAE\anythingKlF8Anime2VaeFtMse840000_vaeFtMse840000.safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Applying attention optimization: sdp... done.
Startup time: 11.7s (prepare environment: 10.6s, initialize shared: 1.6s, load scripts: 3.5s, create ui: 2.6s, gradio launch: 0.6s).
Model loaded in 4.4s (load weights from disk: 0.2s, create model: 0.8s, apply weights to model: 1.4s, load VAE: 0.6s, load textual inversion embeddings: 0.3s, calculate empty prompt: 0.9s).
```

**Comment by BlafKing** [comment]
This is unfortunately an issue caused by the CivitAI API.

I've notified the CivitAI Team about the issue, so all we can do is wait for now.
You can see the issue tracker for it [here](https://github.com/civitai/civitai/issues/1047)

**Comment by Kaleeb44** [comment]
i figured it would be something like that.
Thanks for the quick update!

**Comment by BlafKing** [comment]
This issue is now fixed by an API update, but this also caused this extension to no longer work.

I've started working on a fix! #229
I'll close this bug report as the issue related to this bug is now fixed 👍🏻
